index.ejs

```diff
@@ -66,7 +67,7 @@
 <% } %>
 
 <% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
-<script src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
+<script src="<%=prependUrl%><%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
```

Comment:
Man thank you so much, maybe I was in such a hurry that I fogot publicPath :( thank you my man and Im gonna close the pull request 

THANKS SO MUCH !

I will make another PR To update readme.... Give me a minute :D lol At least I tried it !